code/modules/power/pt_laser.dm

```diff
@@ -465,25 +465,25 @@
 
 	//this will probably need fiddling with, hard to decide on reasonable values
 	switch(power)
-		if(10 to 1e7)
+		if(10 to 10 MEGA WATTS)
```

Comment:
should this be to 5 MEGA WATTS?

Reply:
Good catch, you really would think the compiler should be able to catch this.